### PR TITLE
Release 27.1.1snap1

### PR DESCRIPTION
**Pull request by kyrofa**
- mysql: revert BUG#34849343 Aligned_atomic not working as intended
- nextcloud: update to 27.1.1

### CODE REVIEW

File: .github/workflows/launchpad.yml [hunk 1]
```diff
@@ -2,11 +2,13 @@ name: Deploy to Launchpad
...
 on:
   push:
-    # Only applies to our protected branches
+    # Only applies to our protected branches and tags
     branches:
       - master
       - develop
       - '[0-9]+'
+    tags:
+      - '*'
```
Comment (Pilzinsel64) [inline .github/workflows/launchpad.yml:11]: Uh nice! That will push to launchpad also when we create a tag for the previously created merge commit, right? :)

**Review by Pilzinsel64 — APPROVED** [review]
👍

File: .github/workflows/launchpad.yml (hunk 1 above)
Comment (kyrofa) [reply .github/workflows/launchpad.yml:11]: Yes that's correct.

Comment (kyrofa) [reply .github/workflows/launchpad.yml:11]: Agh, this doesn't actually work due to https://github.com/github/roadmap/issues/825 .

**Pull request merged**